temporalio/worker/_worker.py

```diff
@@ -94,6 +137,12 @@ def __init__(
         use_worker_versioning: bool = False,
         disable_safe_workflow_eviction: bool = False,
         deployment_config: Optional[WorkerDeploymentConfig] = None,
+        workflow_task_poller_behavior: PollerBehavior = PollerBehaviorSimpleMaximum(
+            maximum=5
+        ),
```

Comment:
May be clearer if there is a `default` `ClassVar` on the dataclass that represents the default, e.g. `PollerBehaviorSimpleMaximum.default` instead of inline code, but not required

In fact, looking at params here, if you fix the issue where you're accidentally ignoring the `max_concurrent_workflow_task_polls` and `max_concurrent_activity_task_polls` now, you're going to need to know whether one of the other is set, so you may need a arg-unset sentinel type of value here instead of a valuable default.